src/less/tables.less

```diff
@@ -698,3 +698,59 @@
   }
 }
 
+@media (max-width: @global-screen-small-max) {
+
+  .ReactTable {
```

Comment:
Are we using capital letters for class names anywhere else?

Reply:
Actually, these classes are generated by React Table and as per design, I wanted to overwrite few css rules for these. @alexgsummer